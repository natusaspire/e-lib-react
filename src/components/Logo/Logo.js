import React, { memo } from 'react';

const Logo = () => <div className="logo">E-lib</div>;

export default memo(Logo);
