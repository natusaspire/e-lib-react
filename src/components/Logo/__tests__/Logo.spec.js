import React from 'react';
import { shallow } from 'enzyme';

import Logo from '@/components/Logo';

describe('<Logo />', () => {
  it('should render correctly', () => {
    const wrapper = shallow(<Logo />);

    expect(wrapper).toMatchSnapshot();
  });
});
