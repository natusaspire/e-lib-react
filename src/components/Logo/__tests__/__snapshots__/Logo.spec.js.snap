// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Logo /> should render correctly 1`] = `
<div
  className="logo"
>
  E-lib
</div>
`;
